src/pages/test-report/cat-c/components/reverse-left/reverse-left.ts

```diff
@@ -0,0 +1,115 @@
+import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs/Subscription';
+import { Store, select } from '@ngrx/store';
+import { StoreModel } from '../../../../../shared/models/store.model';
+import { FaultCountProvider } from '../../../../../providers/fault-count/fault-count';
+import { getTests } from '../../../../../modules/tests/tests.reducer';
+import { getCurrentTest } from '../../../../../modules/tests/tests.selector';
+
+// TODO: MES-4287  Import cat c reducer
+import { getTestData } from '../../../../../modules/tests/test-data/cat-be/test-data.cat-be.reducer';
+
+// TODO: MES-4287 Import cat c selector
+import { getManoeuvres } from '../../../../../modules/tests/test-data/cat-be/test-data.cat-be.selector';
+
+// TODO: MES-4287 Import cat C schema
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { TestCategory } from '@dvsa/mes-test-schema/categories/common/test-category';
+import { CompetencyOutcome } from '../../../../../shared/models/competency-outcome';
+import { OverlayCallback } from '../../../test-report.model';
+import { ReverseLeftPopoverOpened, ReverseLeftPopoverClosed } from './reverse-left.actions';
+import { RecordManoeuvresSelection } from '../../../../../modules/tests/test-data/common/manoeuvres/manoeuvres.actions';
+import { ManoeuvreTypes } from '../../../../../modules/tests/test-data/test-data.constants';
+
+// TODO: MES-4287 Import cat c selector
+import { getReverseLeftSelected }
+  from '../../../../../modules/tests/test-data/cat-be/manoeuvres/manoeuvres.cat-be.selectors';
+
+// TODO: MES-4287 Import cat c actions
+import { DeselectReverseLeftManoeuvre }
+  from '../../../../../modules/tests/test-data/cat-be/manoeuvres/manoeuvres.cat-be.actions';
+
+@Component({
+  selector: 'reverse-left',
+  templateUrl: 'reverse-left.html',
+})
+export class ReverseLeftComponent implements OnInit, OnDestroy  {
+
+  @Input()
+  completed: boolean;
+
+  @Input()
+  controlLabel: string;
+
+  @Input()
+  clickCallback: OverlayCallback;
+
+  completedReverseLeft: boolean = false;
+
+  drivingFaults: number = 0;
+  hasSeriousFault: boolean = false;
+  hasDangerousFault: boolean = false;
+
+  subscription: Subscription;
+
+  displayPopover: boolean = false;
+
+  constructor(
+    private store$: Store<StoreModel>,
+    private faultCountProvider: FaultCountProvider,
+  ) {}
+
+  ngOnInit(): void {
+    const manoeuvres$ = this.store$.pipe(
+      select(getTests),
+      select(getCurrentTest),
+      select(getTestData),
+      select(getManoeuvres),
+    );
+
+    this.subscription = manoeuvres$.subscribe((manoeuvres: CatBEUniqueTypes.Manoeuvres) => {
```

Comment:
TODO comments about Cat C types would be nice.

Reply:
added the TODO